lib/internal/cluster/primary.js

```diff
@@ -152,10 +152,11 @@ function removeWorker(worker) {
 function removeHandlesForWorker(worker) {
   assert(worker);
 
-  handles.forEach((handle, key) => {
+  // eslint-disable-next-line node-core/no-array-destructuring
+  for (const [key, handle] of handles) {
```

Comment:
The lint rule is there for a reason
```suggestion
  for (const { 0: key, 1: handle } of handles) {
```